src/components/Editor/SearchBar.js

```diff
@@ -640,6 +666,10 @@ const SearchBar = React.createClass({
         onChange: this.onChange,
         onKeyUp: this.onKeyUp,
         onKeyDown: this.onKeyDown,
+        handleNext: e =>
```

Comment:
hmm, not sure if we need the ternary. if we're not rendering the buttons will these be used?